### DISCUSSION TITLE
logging with debug level

### DISCUSSION THREAD

**Discussion by nouaman1** [Keyfactor/ejbca-ce#36] [Q&A] [answered]
I am running EJBCA using the official Docker image.
I want to understand why some errors happen. 
I see the default logging level is INFO.
How can I change it to level DEBUG?
Thanks.

**Comment by pkjohan** [comment]
Hi nouaman1,

Try adding the the environment variable `DEBUG=true` to your deployment (e.g. `--env DEBUG=true` for `docker`/`podman`)

BR,
J

**Reply by nouaman1** [reply]
Hi pkjohan,

Thank you for your quick reply.
I am using this Command to ran the docker:
docker run -it --rm -p 80:8080 -p 443:8443 -h localhost -e TLS_SETUP_ENABLED="simple" primekey/ejbca-ce
When i add --env DEBUG=true i get this error:
 docker run -it --rm -p 80:8080 -p 443:8443 -h localhost -e TLS_SETUP_ENABLED="simple" primekey/ejbca-ce --env DEBUG=true
docker: Error response from daemon: OCI runtime create failed: container_linux.go:380: starting container process caused: exec: "--env": executable file not found in $PATH: unknown.

How can i add the environment variable DEBUG=true correcktly?

BR

**Comment by nouaman1** [comment, marked as answer]
this command did the job:

docker run -it --rm -p 80:8080 -p 443:8443 -h localhost -e TLS_SETUP_ENABLED="simple" -e LOG_LEVEL_APP=DEBUG primekey/ejbca-ce 

BR
